library/ssl_tls.c

```diff
@@ -4650,8 +4650,15 @@ const char *mbedtls_ssl_get_version( const mbedtls_ssl_context *ssl )
 
     switch( ssl->minor_ver )
     {
+#if defined(MBEDTLS_SSL_PROTO_TLS1_2)
         case MBEDTLS_SSL_MINOR_VERSION_3:
             return( "TLSv1.2" );
+#endif /* MBEDTLS_SSL_PROTO_TLS1_2 */
+
+#if defined(MBEDTLS_SSL_PROTO_TLS1_3)
```

Comment:
With the merge of #5426, you should not have to do that anymore.

Reply:
removed from this PR